src/structs.md

```diff
@@ -17,3 +17,14 @@ fn main() {
     println!("{} is {} years old", peter.name, peter.age);
 }
 ```
+
+<details>
+Key Points: 
+
+* Structs work similar as they do in other languages, but note there is no `typedef` keyword or inheritance between structs. 
+* Methods are defined in an `impl` block, which we will see in following slides.
+* This may be a good time to let people know there are different types of structs. The next slide will introduce Tuple structs. Here you can define a unit-like struct.
+    * Unit-like structs can be defined with `struct (name);`. There is no need for curly brackets and parentheses.
```

Comment:
The bullet point about different types could also be split up into a list.

I had trouble understanding the line with units and `struct (name)` (first thought about the newtype pattern. I'd suggest either mentioning in a new bullet point that structs can be empty (zero-sized structs) and then either not showing an example, or using a metasyntactic name, e.g., `struct Foo;`.